airbyte-integrations/bases/base-normalization/normalization/destination_type.py

```diff
@@ -27,5 +27,7 @@
 
 class DestinationType(Enum):
     bigquery = "bigquery"
+    jdbc_postgres = "jdbc_postgres"
```

Comment:
Why do we have both?

Reply:
I mirrored what we have in the source-jdbc (which is a source-postgres in the end)

This option here in the normalization is necessary to run tests/integrations tests on the destination-jdbc code with normalization.

But since we don't expose neither destination-jdbc nor source-jdbc in the UI, it shouldn't get any usage other than tests...

Reply:
@cgardens should we get rid of JdbcDestination then ? (and JdbcSource too?)

Reply:
We want to be able to run unit tests (and depending on your judgment integration tests) on the Jdbc version independent of the the other destinations that inherit it. I think that can probably be done without including it in the normalization code.